dockerfiles/Dockerfile

```diff
@@ -10,17 +10,31 @@ RUN apk --no-cache upgrade && \
 	openssl-dev \
 	libxml2-dev \
 	libxslt-dev \
+	bash \
+	sudo \
+	openssh \
+	tar \
+	acl \
+	g++ \
+	git \
 	curl && \
 	pip install --upgrade pip && \
 	rm -rf /tmp/* && \
-	rm -rf /var/cache/apk/*
+	rm -rf /var/cache/apk/* && \
+	ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa && \
+	mkdir -p /var/run/sshd && \
+	mkdir -p /root/.ssh && \
+	chmod 0700 /root/.ssh && \
+	passwd -u root && \
+	pip install git+git://github.com/esnme/ultrajson.git
```

Comment:
Does it necessary to install via git? we have `ujson` in `requirements.txt` already.

Reply:
https://github.com/esnme/ultrajson/issues/326

Reply:
Thanks for the info.